test/dot_s/basics.s

```diff
@@ -48,7 +48,7 @@ main:                                   # @main
 	end_block
 	i32.const	$push11=, -12
 	i32.add 	$0=, $0, $pop11
-	i32.const	$discard=, main # just take address for testing
+	i32.const	$discard=, main@FUNCTION # just take address for testing
```

Comment:
The wasm backend output changed a long time ago, it always appends `@FUNCTION` to function names in contexts like this (see e.g. all of the other dot_s tests and waterfall output). It just so happened that before s2wasm didn't care about it.